modules/perception/onboard/component/fusion_camera_detection_component.cc

```diff
@@ -798,6 +812,42 @@ int FusionCameraDetectionComponent::MakeProtobufMsg(
       return cyber::FAIL;
     }
   }
+
+  // write out lanes in ego coordinates
+  apollo::perception::LaneMarkers *lane_markers =
+      obstacles->mutable_lane_marker();
+  apollo::perception::LaneMarker *lane_marker_l0 =
+      lane_markers->mutable_left_lane_marker();
+  apollo::perception::LaneMarker *lane_marker_r0 =
+      lane_markers->mutable_right_lane_marker();
+  apollo::perception::LaneMarker *lane_marker_l1 =
+      lane_markers->add_next_left_lane_marker();
+  apollo::perception::LaneMarker *lane_marker_r1 =
+      lane_markers->add_next_right_lane_marker();
+
+  for (const auto &lane : lane_objects) {
+    base::LaneLineCubicCurve curve_coord = lane.curve_image_coord;
+    if (&lane.curve_car_coord.a != nullptr) {
```

Comment:
This seems true forever.

Reply:
Can you suggest a way to check whether this struct has been assigned values?

Reply:
Never mind, I will modify lane detection to make sure there are always valid outputs in curve_car_coord